src/Illuminate/Foundation/Auth/AuthenticatesUsers.php

```diff
@@ -88,7 +89,7 @@ protected function attemptLogin(Request $request)
      */
     protected function credentials(Request $request)
     {
-        return $request->only($this->username(), 'password');
+        return $request->only($this->username(), $this->password());
```

Comment:
These credentials must have a field named password that contains the plain text password. This is the only way the User Providers know which field is the actual password so it doesn't add the password field to the where query and also which field to do the hash check on after a result comes back for the rest of the credentials. It is hardcoded that way.